projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.ts

```diff
@@ -41,6 +41,7 @@ export class TimeSeriesChartComponent implements OnInit, AfterViewInit, OnChange
 
   @Output() onZoomReset = new EventEmitter<void>();
   @Output() onZoomChange = new EventEmitter<TSTimeRange>();
+  @Output() onLockStateChange = new EventEmitter<boolean>();
```

Comment:
https://angular.io/guide/styleguide#style-05-16
Don't name events with `on` prefix.
Linter should warned about it. I didn't configured it as error, because unfortunately there are several places, where we such names still exists. But we should avoid of usage of such event names